VM/plot_vm.py

```diff
@@ -113,30 +134,38 @@ def plot_vm(
         background="white",
         out_name=(outdir / vm_params_dict["name"]).resolve(),
     )
+    return passed
```

Comment:
Isn't this always going to return True? What does this achieve

Reply:
Oops. Removed